_component_design/time-selector.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Time Selector
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: The time selector enables the user to select a time from a list of values. The time selector is often accompanied by a date selector that a user can use to select a date.
+preview-image: preview-images/time-selector.svg
+resource: true
+status: stable
+last-modified: 2018-11-20
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the time selector to enable a user to choose a specific hour in the 24-hour range. The time selected includes an hour value and a minute value.
+
+If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}\components\radios.html).
```

Comment:
```suggestion
If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}/components/radio-buttons.html).
```

Reply:
Weird, I coulda sworn I changed the name of radio-buttons to just radios. Looks like I did not do that. Accepted.